website/src/pages/index.tsx

```diff
@@ -1,16 +1,139 @@
-import { HeroGradient, InfoList } from '@theguild/components'
+import { HeroGradient, HeroIllustration } from '@theguild/components'
 import { handlePushRoute } from '@guild-docs/client'
+import { Component, MouseEvent, useMemo } from 'react'
+import { ILink } from '@theguild/components/dist/types/components'
+import { LinkProps } from 'next/link'
+import { theme } from '@chakra-ui/react'
+import styled from '@emotion/styled'
+import { CssTheme } from '@chakra-ui/styled-system/dist/declarations/src/utils'
+
+const Image = styled.img`
+  margin-right: 20px;
+`
+
+interface featuresOptions {
+  title: string
+  description: string
+  imageSrc: string
+  imageAlt: string
+  flipped?: boolean
+  link?: ILink
+}
 
 export default function Index() {
+  const ITEMS = useMemo<featuresOptions[]>(
+    () => [
+      {
+        title: 'GraphQL over HTTP compliant',
+        description:
+          'GraphQL over HTTP specification-compliant server. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Diam suscipit aliquet id curabitur in at.',
+        imageSrc: '/assets/http.svg',
+        imageAlt: 'Server over HTTP',
+        link: {
+          children: 'Learn more',
+          title: 'Learn more',
+          href: '/docs',
+          onClick: (e) => handlePushRoute('/docs', e),
+          style: {
+            color: 'white',
+            display: 'flex',
+            flexDirection: 'column',
+            alignItems: 'center',
+            width: '144px',
+            height: '40px',
+            borderRadius: '6px',
+            backgroundColor: '#1CC8EE',
+            justifyContent: 'center',
+          },
+        },
+      },
+      {
+        title: 'Extensible Engines with Envelop Ecosystem',
+        description:
+          'Extend GraphQL Engines capabilities with Envelops powerful plugin ecosystem. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Diam suscipit aliquet id curabitur in at.',
+        imageSrc: '/assets/ecosystem.svg',
+        imageAlt: 'Ecosystem',
+        flipped: true,
+        link: {
+          children: 'Learn more',
+          title: 'Learn more',
+          href: '/docs',
+          onClick: (e) => handlePushRoute('/docs', e),
+          style: {
+            color: 'white',
+            display: 'flex',
+            flexDirection: 'column',
+            alignItems: 'center',
+            width: '144px',
+            height: '40px',
+            borderRadius: '6px',
+            backgroundColor: '#1CC8EE',
+            justifyContent: 'center',
+          },
+        },
```

Comment:
Are we planning to take to some more detailed docs? If not for now we can just hide these `Learn more` buttons until we get more docs.

Reply:
No problem :)